### DISCUSSION TITLE
Swift AppDelegate support

### DISCUSSION THREAD

**Discussion by jclif** [agencyenterprise/react-native-health#307] [General]
Is there a way to add background task support with Swift AppDelegate files?

**Comment by sala7khaled** [comment]
Yes, you can add background task support in Swift using the AppDelegate file. The AppDelegate class in Swift is responsible for handling the application's lifecycle events, including background tasks.

To add background task support, you'll need to use the `beginBackgroundTask(withName:expirationHandler:)` method provided by `UIApplication.shared`.
